scripts/rpm/post-install.sh

```diff
@@ -84,5 +84,18 @@ elif [[ -f /etc/os-release ]]; then
     elif [[ "$NAME" = "Solus" ]]; then
         # Solus logic
         install_systemd /usr/lib/systemd/system/telegraf.service
+    elif [[ "$NAME" = "SLES" ]]; then
+        if [[ "$(readlink /proc/1/exe)" == */systemd ]]; then
+            install_systemd /usr/lib/systemd/system/telegraf.service
+        else
+            # Assuming SysVinit
+            install_init
+            # Run update-rc.d or fallback to chkconfig if not available
+            if which update-rc.d &>/dev/null; then
+                install_update_rcd
+            else
+                install_chkconfig
+            fi
+        fi
```

Comment:
This is copied from line 58.  Would you put it in a function and call it from both places?  Maybe name the function install_systemd_or_chkconfig

Reply:
Yes, of course. Added a function in a new commit